chia/full_node/mempool_manager.py

```diff
@@ -158,7 +163,9 @@ def __init__(
         self.nonzero_fee_minimum_fpc = 5
 
         BLOCK_SIZE_LIMIT_FACTOR = 0.7
-        self.max_block_clvm_cost = uint64(self.constants.MAX_BLOCK_COST_CLVM * BLOCK_SIZE_LIMIT_FACTOR)
+        BLOCK_OVERHEAD = QUOTE_BYTES * self.constants.COST_PER_BYTE + QUOTE_EXECUTION_COST
+
+        self.max_block_clvm_cost = uint64(self.constants.MAX_BLOCK_COST_CLVM * BLOCK_SIZE_LIMIT_FACTOR - BLOCK_OVERHEAD)
```

Comment:
are you sure this is the right variable to deduct the block overhead cost from?
In your first version of the patch you, effectively, altered `self.mempool_info.max_block_clvm_cost`

Reply:
Yes, `mempool_info` uses this value for its `max_block_clvm_cost`: https://github.com/Chia-Network/chia-blockchain/blob/9eb07b1fef96a3a84e3be7d62535b397d95581b9/chia/full_node/mempool_manager.py#L184-L188